src/core/js/views/questionView.js

```diff
@@ -279,6 +282,23 @@ define([
         // be a full reset
         resetQuestion: function() {},
 
+        refresh: function() {
+          this.renderState();
+          
+          this._runModelCompatibleFunction("updateButtons");
+
+          if (this.model.get('_canShowMarking') && this.model.get('_isInteractionComplete') && this.model.get('_isSubmitted')) {
+            this.showMarking();
+          }
+
+          if (this.buttonsView) {
+            _.defer(_.bind(function() {
+                this.buttonsView.refresh();
+              }, this)
```

Comment:
May I retract this comment – realise now you're needing to actually call the function at this point! Apologies.